src/main.cpp

```diff
@@ -350,8 +356,30 @@ void radarLoop()
     }
 }
 
+
+void dhtLoop()
+{
+    if (!dht11Pin && !dht22Pin) return ;
+    delay(dht.getMinimumSamplingPeriod());
```

Comment:
loop shouldn't have a delay like this because all the other things need their turn.  Instead keep track of the millis since you updated in a variable and once you've gotten > the sampling period then update and save the last updated